image/alpine3/Dockerfile

```diff
@@ -23,8 +23,7 @@ RUN apk update && \
         python \
         linux-headers \
         binutils-gold \
-        libstdc++ && \
-    rm -rf /var/cache/apk/*
+        libstdc++
```

Comment:
would that mean that we don't ship a compiler? I think that is very much needed for any Node.js application. Adding  it later on just increase the build time as these things would need to be installed at build time.

Reply:
I am trying to find out where the extra ~100mb is coming from, hence WIP. CircleCI won't build anything if it is not a PR.

Reply:
I think clang might be the culprit: https://pkgs.alpinelinux.org/package/edge/main/x86/clang 
`Installed size | 149.18 MB`

Reply:
Now I understand!

Reply:
@mcollina 
1. I got it down to 91mb 
2. the official node image deletes all build deps: `&& apk del .build-deps \` so this gives us a reason and an advantage we should mention
3. in the current state, only removing `clang` I can successfully install sqlite3 from source: `npm install --build-from-source sqlite3`. I will see if there are any other packages  we can do without, like findutils, gettext. 

is installing sqlite3 a good test for compiling native node modules?